updates/pg_partman--4.7.3--5.0.0-beta.sql

```diff
@@ -5462,6 +5452,54 @@ END;
 $$;
 
 
+CREATE PROCEDURE @extschema@.run_maintenance_proc(
+    p_wait int DEFAULT 0
+    -- Keep these defaults in sync with `run_maintenance`!
+    , p_analyze boolean DEFAULT false
+    , p_jobmon boolean DEFAULT true
+)
+    LANGUAGE plpgsql
+    AS $$
+DECLARE
+
+v_adv_lock              boolean;
+p_parent_table          text;
+
+BEGIN
+
+v_adv_lock := pg_try_advisory_lock(hashtext('pg_partman run_maintenance'));
+IF v_adv_lock = false THEN
+    RAISE NOTICE 'Partman maintenance already running or another session has not released its advisory lock.';
+    RETURN;
+END IF;
+
+FOR p_parent_table IN
```

Comment:
```suggestion
FOR v_parent_table IN
```
Please use `v_` prefix for variables that aren't parameters